src/components/application_manager/test/commands/mobile/add_command_request_test.cc

```diff
@@ -0,0 +1,195 @@
+#include <stdint.h>
+#include <string>
+#include <set>
+
+#include "gtest/gtest.h"
+#include "utils/shared_ptr.h"
+#include "utils/helpers.h"
+#include "utils/make_shared.h"
+#include "smart_objects/smart_object.h"
+#include "interfaces/MOBILE_API.h"
+#include "utils/custom_string.h"
+#include "application_manager/mock_application.h"
+#include "application_manager/commands/command_request_test.h"
+#include "application_manager/commands/mobile/add_command_request.h"
+#include "application_manager/smart_object_keys.h"
+#include "application_manager/commands/commands_test.h"
+#include "application_manager/commands/command_impl.h"
+#include "application_manager/application_manager.h"
+#include "policy/usage_statistics/mock_statistics_manager.h"
+#include "application_manager/mock_application.h"
+#include "application_manager/mock_application_manager.h"
+#include "application_manager/mock_message_helper.h"
+#include "application_manager/message_helper.h"
+#include "application_manager/event_engine/event.h"
+
+
+namespace test {
+namespace components {
+namespace commands_test {
+
+namespace am = application_manager;
+using am::commands::CommandImpl;
+using am::ApplicationManager;
+using am::commands::MessageSharedPtr;
+using am::ApplicationSharedPtr;
+using am::MockMessageHelper;
+using ::testing::_;
+using ::utils::SharedPtr;
+using ::testing::Return;
+
+
+using ::test::components::application_manager_test::MockApplication;
+using usage_statistics_test::MockStatisticsManager;
+using am::event_engine::EventObserver;
+
+
+namespace smart_objects = NsSmartDeviceLink::NsSmartObjects;
+namespace custom_str = utils::custom_string;
+namespace strings = ::application_manager::strings;
+namespace hmi_response = ::application_manager::hmi_response;
+
+class AddCommandRequestImplTest
+    : public CommandRequestTest<CommandsTestMocks::kIsNice> {
+};
+
+const hmi_apis::FunctionID::eType kInvalidFunctionId_ =
+    hmi_apis::FunctionID::INVALID_ENUM;
+
+TEST_F(AddCommandRequestImplTest,  GetRunMethods_SUCCESS){
+//#ifdef __UNIT_TEST__
+//     friend AddCommandRequest;
+//   #endif
+     MessageSharedPtr msg = CreateMessage(smart_objects::SmartType_Map);
+
+     smart_objects::SmartObject menu_params = smart_objects::SmartObject(smart_objects::SmartType_Map);
+     menu_params[am::hmi_request::parent_id] = 1;
+     menu_params[strings::position] = 10;
+     menu_params[strings::menu_name] = "LG";
+
+
+     smart_objects::SmartObject msg_params = smart_objects::SmartObject(smart_objects::SmartType_Map);
+     msg_params[strings::cmd_id] = 1;
+     msg_params[strings::menu_params] = menu_params;
+     msg_params[strings::app_id] = 34;
+    (*msg)[strings::msg_params] = msg_params;
+
+
+     using namespace application_manager::commands;
```

Comment:
@wolfylambova22, you are using it at every test and just to get `AddCommandRequest`. Don't it would be better to just  define usage of `AddCommandRequest` at the beginning of the file? 
> `using am::commands::AddCommandRequest`